modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -202,7 +202,7 @@ Otherwise it is matched as a case-sensitive string.
 
 | `features`
 | `array (string)`
-| List of additional features to enable. They are all disabled by default. Enabling additional features might have performance impacts. A possible value is `+`.
+| List of additional features to enable. They are all disabled by default. Enabling additional features might have performance impacts. Possible values are: +
```

Comment:
Do we need the `+` at the end? It doesn't seem to be appearing in the preview. Not sure what it does.